src/main/java/cubicchunks/VanillaCubicChunksWorldType.java

```diff
@@ -49,10 +50,12 @@ public VanillaCubicChunksWorldType() {
 		// init the worldgen pipeline
 		GeneratorStage terrain = new GeneratorStage("terrain");
 		GeneratorStage surface = new GeneratorStage("surface");
+		GeneratorStage testStage = new TestStage("testStage");
```

Comment:
Is this actually needed or was it just for testing? If it's just for testing, it should be removed.